Resources/public/css/theme/views/fields/edit/country.css

```diff
@@ -0,0 +1,10 @@
+.ez-view-countryeditview .ez-field-description {
```

Comment:
(and the copyright header is missing on the new CSS files, if some new CSS need to be kept)